src/main/java/org/truffleruby/options/CommandLineParser.java

```diff
@@ -557,8 +545,12 @@ private String grabOptionalValue() {
         return null;
     }
 
+    private CommandLineException notImplemented(String option) {
+        return new CommandLineException(String.format("the %s option is not implemented", option));
+    }
+
     public static void printHelp(PrintStream out) {
-        out.println("Usage: truffleruby [switches] [--] [programfile] [arguments]");
+        out.println("Usage: ruby [switches] [--] [programfile] [arguments]");
```

Comment:
Are you talking about just the help and usage output, or the prefix on error messages as well?